challenges/objects-arrays.js

```diff
@@ -52,6 +79,21 @@ const graduates = [
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for (let i = 0; i < graduates.length; i++) {
+  let mappedObj = {};
+  mappedObj.university = graduates[i].university;
+  universities.push(mappedObj);
+}
+universities.sort(function (a, b) {
```

Comment:
this does work, and i'm impressed, but .sort on its own would sort alphabetically without all that logic :wink: